docs/user/tutorials/01-11-manage-unmanage-modules.md

```diff
@@ -0,0 +1,32 @@
+# Setting Your Module to the Managed/Unmanaged State in Kyma CLI
+
+In some cases, for example, for testing, you may need to modify your module beyond what is supported by its configuration. By default, when a module is in the managed state, Kyma Control Plane governs its Kubernetes resources, reverting any manual changes during the next reconciliation loop. 
+
+To modify Kubernetes objects directly without them being reverted, you must set the module to the unmanaged state. In this state, reconciliation is disabled, ensuring your manual changes are preserved.
+
+> [!CAUTION]
+> Setting your module to the Unmanaged state may lead to instability and data loss within your cluster. It may also be impossible to revert the changes. In addition, we don't guarantee any service level agreement (SLA) or provide updates and maintenance for the module.
```

Comment:
```suggestion
> Setting your module to the unmanaged state may lead to instability and data loss within your cluster. It may also be impossible to revert the changes. In addition, we don't guarantee any service level agreement (SLA) or provide updates and maintenance for the module.
```